department.rb

```diff
@@ -0,0 +1,36 @@
+class Department
+  attr_accessor :name, :employees
+  def initialize(name)
+    @employees = []
+    @name = name
+  end
+
+  def add(employee)
+    employee.department = @name
```

Comment:
This is fine (I like that you want to make a double reference.  Curious why you didn't do `employee.department = self`, though.  It would seem as though having access to the whole department rather than just its name could be useful later on.